### PR TITLE
Made a few updates regarding storage methods and order_by

### PR DESCRIPTION
**Pull request by symbiont-haris-amin**
* Note the second param of `order_by` species `ASCENDING` order, removing the misleading and incorrect `desc=True` arg
* cleaned up `_get_rooms` a little

### CODE REVIEW

File: chat.sympl [hunk 1]
```diff
@@ -525,50 +525,27 @@ def _get_messages(room_channel: ChannelName) -> List[Message]:
...
-        return cvm.storage.query_history(MessageStatic).in_channel(room_channel).order_by('timestamp', desc=True).values()
+        return cvm.storage.query_history(MessageStatic).in_channel(room_channel).order_by('timestamp', False).values()
     else:
         cvm.error(f"Room for channel {room_channel} not found.")
 
 @clientside_helper
 def _get_rooms() -> List[Room]:
     ##this function gets the most recent readable version of the room by the caller (i.e. this gets the current version
     # if you are still in the room, and if you were kicked out, the version immediately after you were kicked out)
+    results = cvm.storage.query_history(RoomStatic).where('is_deleted', False).order_by('channel', False).order_by('name', False).values()
```
Comment (symbiont-riccardo-abbate) [inline chat.sympl:536]: the original code had an explicit type cast for the variable `results`. Why did we remove it?

**Review by symbiont-riccardo-abbate — COMMENTED** [review]
I would like someone to confirm that the whole set of tests for the contract and for the middleware has been run.

File: chat.sympl [hunk 2]
```diff
@@ -525,50 +525,27 @@ def _get_messages(room_channel: ChannelName) -> List[Message]:
     if isinstance(room, Room):
         if room.is_deleted:
             cvm.error(f"Room {room_channel} has been deleted. Cannot get messages.")
-        return cvm.storage.query_history(MessageStatic).in_channel(room_channel).order_by('timestamp', desc=True).values()
+        return cvm.storage.query_history(MessageStatic).in_channel(room_channel).order_by('timestamp', False).values()
```
[the remark this answers is not in the document]
Comment (symbiont-haris-amin) [reply chat.sympl]: It actually wasn't using `desc=True`. `order_by`'s second arg always species whether `ASCENDING` order is `True/False`. The label `desc=` in the `kwarg` was being ignored and shouldn't work at all. Its misleading at the moment and the language team is aware of it.

**Comment by symbiont-haris-amin** [comment]
> I would like someone to confirm that the whole set of tests for the contract and for the middleware has been run.

@symbiont-riccardo-abbate i just ran the tests for the contract and they all pass. @symbiont-david-bono can u confirm that the middleware tests pass please?

**Review by symbiont-david-bono — APPROVED** [review]
LGTM

**Pull request merged**